test/unit/configuration/normalizeConfig-test.js

```diff
@@ -0,0 +1,218 @@
+          assert.equal(_utils.coerceToBoolean(''), false);
+        });
+      });
+    });
+
+    describe('c (color alias)', () => {
+      const result = _utils.coerceColorOption({ c: false });
+
+      it('coerces to boolean "color" option', () => {
+        assert.propertyVal(result, 'color', false);
+      });
+
+      it('removes "c" option', () => {
+        assert.notProperty(result, 'c');
+      });
+    });
+
+    describe('apiDescriptions', () => {
+      describe('when given a string', () => {
+        const result = _utils.coerceApiDescriptions('foo');
+
+        it('coerces into list of descriptions', () => {
+          assert.deepEqual(result, [
+            {
+              location: 'configuration.apiDescriptions[0]',
+              content: 'foo',
+            },
+          ]);
+        });
+      });
+
+      describe('when given a list', () => {
+        const result = _utils.coerceApiDescriptions(['foo', 'bar']);
+
+        it('coerces into list of descriptions', () => {
+          assert.deepEqual(result, [
+            { location: 'configuration.apiDescriptions[0]', content: 'foo' },
+            { location: 'configuration.apiDescriptions[1]', content: 'bar' },
+          ]);
+        });
+      });
+    });
+
+    describe('user', () => {
+      const result = _utils.coerceUserOption({ user: 'apiary' });
+
+      it('coerces to base64 encoded "header"', () => {
+        assert.deepEqual(result.header, ['Authorization: Basic YXBpYXJ5']);
+      });
+
+      it('removes "user" option', () => {
+        assert.notProperty(result, 'user');
+      });
+    });
+  });
+
+  describe('coercion of deprecated options', () => {
+    describe('level', () => {
+      describe('coerces to "debug"', () => {
+        it('when given "silly"', () => {
+          assert.propertyVal(_utils.coerceDeprecatedLevelOption({ l: 'silly' }), 'loglevel', 'debug');
+        });
+
+        it('when given "verbose"', () => {
+          assert.propertyVal(_utils.coerceDeprecatedLevelOption({ l: 'verbose' }), 'loglevel', 'debug');
+        });
+
+        it('when given "debug"', () => {
+          assert.propertyVal(_utils.coerceDeprecatedLevelOption({ l: 'debug' }), 'loglevel', 'debug');
+        });
+      });
+
+      describe('coerces to "error"', () => {
+        it('when given "error"', () => {
+          assert.propertyVal(_utils.coerceDeprecatedLevelOption({ l: 'error' }), 'loglevel', 'error');
+        });
+      });
+
+      describe('coerces to "silent"', () => {
+        it('when given "silent"', () => {
+          assert.propertyVal(_utils.coerceDeprecatedLevelOption({ l: 'silent' }), 'loglevel', 'silent');
+        });
+      });
+
+      describe('coerces to "warn"', () => {
+        it('in all other cases', () => {
+          assert.propertyVal(_utils.coerceDeprecatedLevelOption({ l: null }), 'loglevel', 'warn');
+          assert.propertyVal(_utils.coerceDeprecatedLevelOption({ l: null }), 'loglevel', 'warn');
```

Comment:
These two are identical lines. Perhaps you wanted to test `null`, `warn`, `warning`, `foobar`, IDK

Reply:
These should be fixed, probably a stale state of pull request.

Reply:
Yup, you made the commits meanwhile